.pre-commit-config.yaml

```diff
@@ -60,3 +60,11 @@ repos:
     rev: v1.23.0
     hooks:
       - id: yamllint
+  - repo: https://github.com/igorshubovych/markdownlint-cli
```

Comment:
Why do you not use the repo https://github.com/markdownlint/markdownlint? It is more popular than `igorshubovych/markdownlint-cli` with more stars.

Reply:
I have tried `https://github.com/markdownlint/markdownlint` but it requires Ruby. It's not worthy to install Ruby in our dev image just for pre-commit. So i choose markdownlint-cli.